test/sanitizer_suppressions/ubsan

```diff
@@ -80,5 +80,6 @@ implicit-unsigned-integer-truncation:leveldb/*
 implicit-integer-sign-change:crc32c/*
 
 implicit-signed-integer-truncation:script/interpreter.cpp
+implicit-integer-sign-change:primitives/block.h
```

Comment:
what was in here that was causing issues?

Reply:
I don't remember. Probably
```
    static const uint32_t DYNAFED_HF_MASK = 1 << 31; 
```
though maybe other things.